pkg/reconciler/eventtype/controller.go

```diff
@@ -21,14 +21,14 @@ import (
 
 	"knative.dev/pkg/logging"
 
-	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
 	"knative.dev/pkg/configmap"
 	"knative.dev/pkg/controller"
 	"knative.dev/pkg/tracker"
 
+	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
```

Comment:
for here,

https://github.com/knative/eventing/blob/cd07f309450287c4d0594fc67e08ae7f813b5265/pkg/reconciler/eventtype/controller.go#L59

can we use `beta1`?

Reply:
at the moment this is still using v1alpha1.Broker everywhere - this PR is intended only to update the EventType resource. Someone more familiar would have to explain if we ought to be reconciling v1beta1.Broker?

Reply:
Yeah, we're still reconciling with v1alpha1 for Brokers since v1beta1 is lossy. We'll drop v1alpha1 support and let's do the move to v1beta1 as part of that.